files/templates/alertmanager/alertmanager.yaml

```diff
@@ -115,6 +121,10 @@ route:
 receivers:
 - name: root
 
+- name: 'harbor_implementation'
+  slack_configs:
```

Comment:
Do you want to customize the actions?

Reply:
Unsure what to customise them with at this time.

Reply:
Like buttons, and so on,,

Reply:
if you don't needed, that's fine

Reply:
Yeah it's ok for now I think in the future this would be good though